manifests/params.pp

```diff
@@ -1,11 +1,24 @@
 # PHP params class
 #
+# === Parameters
+#
+# [*config_root_override*]
+#   The configuration root directory used in parameter calculation
+#
+# [*php_version_override*]
+#   The version of php used in parameter calculation
+#
 class php::params(
-  $cfg_root = undef, # lint:ignore:parameter_documentation
+  $config_root_override = undef,
+  $php_version_override = undef
```

Comment:
I'm using it without hiera, but as an option perhaps we could just allow for the parameters that consume these overrides to be defined somewhere else (like init or something)?